assets/app/view/game/buy_sell_shares.rb

```diff
@@ -33,6 +33,8 @@ def render
                            .map(&:first).sort_by(&:percent).reverse
                        end
 
+        @reserved_shares = @corporation.reserved_shares
```

Comment:
what is this

Reply:
This game has shares that start reserved, so they aren't buyable. This is part of the code to display those shares (like market / treasury / etc)

Reply:
We already have reserved shares. How is this different?

Reply:
This is just view code. I needed to display the action to use the private ability to trade for a reserved share, so this is needed to get the reserved shares down when rendering those buttons. I just wanted to follow the pattern of grouping all shares that you needed to render buttons for up here and then displaying them in the render methods below.